spec/no_reinstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e -x
+
+! grep "job: install bind" y2log && echo "AUTOYAST OK" "job: install bind" y2log && echo "AUTOYAST OK"
```

Comment:
You are right. That has happened by accident. The funny part is that it working....